docs/Reference/API-Methods.md

```diff
@@ -519,8 +519,8 @@ None
 
 `result` : *string* - Current network ID.
 
-| Network ID | Chain | Network | Description
-|------------|-------|---------|-------------------------------|
+| Network ID | Chain | Network | Description                   |
+|:-----------|:------|:--------|:------------------------------|
```

Comment:
I don't know what these whitespace changes mean

Reply:
My text editor automatically tries to fix the display of markdown tables. I thought I disabled this functionality.